scripts/build.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -r -f build
+mkdir build
+cd build
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)